packages/office-ui-fabric-react/src/components/Sticky/Sticky.types.ts

```diff
@@ -29,6 +29,13 @@ export interface IStickyProps extends React.Props<Sticky> {
    * @defaultvalue true
    */
   isScrollSynced?: boolean;
+
+  /**
+   * It decides the order (top to bottom) in which a Sticky element will be arranged when it is sticky,
+   * compared to other Sticky elements when they are sticky. When Sticky elements become sticky,
+   * they are sorted in ascending order based on this prop.
```

Comment:
What would we do in case of conflict? Eg - 2 sticky components which have the same 'order' ?

Reply:
As long as we are deterministic on what is the behavior in this case, i am fine.

---
In reply to: [297167681](https://github.com/OfficeDev/office-ui-fabric-react/pull/9525#discussion_r297167681) [](ancestors = 297167681)